### ISSUE TITLE
Directive @requires doesn't work correctly

### ISSUE THREAD

**Issue by Santinell** [mercurius-js/mercurius#456]
In one service `HOST` I have such schema:

```
type Host @key(fields: "id") {
  id: ObjectID!
  name: String!
  provider: ObjectID!
  region: ID!
  size: ID!
}
```

and in service `DICT` I have such schema:

```
extend type Host @key(fields: "id") {
  id: ObjectID! @external
  size: ID! @external
  region: ID! @external
  provider: ObjectID! @external
  sizeData: Size! @requires(fields: "size")
  regionData: Region! @requires(fields: "region")
  providerData: Provider! @requires(fields: "provider")
}
```

When I make such request:

```
{
  hosts {
    name
    sizeData {
      cores
    }
  }
}
```
I expect that field `size` will be requested automatically, and I could be able to fetch size data by its id.
But instead of it I don't receive `size` in `DICT` service.

mercurius: 7.3.3
nodejs: 14.16.0

**Comment by mcollina** [comment]
Would you like to send a PR to fix this? That'd be great.

Can you provide a full example to reproduce?

**Comment by Santinell** [comment]
Hi, I made example to reproduce problem:
https://github.com/Santinell/mercurius-bug-requires

I am afraid I don't know where the problem is, to make a PR, I only can guess that it somewhere in gateway

**Comment by simone-sanfratello** [comment]
working on that

**Comment by simone-sanfratello** [comment]
solved and released in [version 7.6.0](https://github.com/mercurius-js/mercurius/releases/tag/v7.6.0)
